src/west/app/main.py

```diff
@@ -833,6 +879,24 @@ def do_run(self, args, ignored):
                          'which may be causing this issue.\n'
                          '  Try running "west update" or fixing the manifest.')
 
+class Alias(WestCommand):
+    # An alias command, it does not run itself
+
+    def __init__(self, cmd, args):
+        super().__init__(cmd, args or '<empty>', f'An alias that expands to: {args}')
+
+        self.args = shlex.split(args)
+
+    def do_add_parser(self, parser_adder):
```

Comment:
Do I get this right?
```suggestion
    # Pseudo-parser that will never actually run except for ".print_help()"
    def do_add_parser(self, parser_adder):
```